rosdep/python.yaml

```
@@ -433,6 +433,16 @@ python:
     yakkety_python3: [python3-dev]
     zesty: [python-dev]
     zesty_python3: [python3-dev]
+python-absl-py-pip:
+  debian:
+    pip:
+      packages: [absl-py]
+    fedora:
```

Looks like there's an indentation error here.
```suggestion
  fedora:
```